📖 Update documentation (#248)
hashicorp/terraform-cloud-operator: docs/examples/README.md

# Examples

> [Here](./) you can find different examples of the YAML manifests. They aim to demonstrate the usage of the different features supported by the Operator and do not seek to demonstrate all possible combinations.

Reviewer: ```suggestion
[Here](./) you can find different examples of the YAML manifests. They aim to demonstrate the usage of the different features supported by the Operator. These examples are not comprehensive, and do not illustrate all possible configurations. 
```
